presto-hive/src/test/java/com/facebook/presto/hive/TestHiveHistoryBasedStatsTracking.java

```diff
@@ -155,13 +157,54 @@ public void testHistoryMatchingThreshold()
         }
     }
 
+    @Test
+    public void testHistoryMatchingSkipInputCheckThreshold()
+    {
+        String tableName = "test_history_matching_threshold";
+        try {
+            assertUpdate("create table " + tableName + "(a int, b varchar)");
+            String query = "select * from " + tableName + " where a <= 2 order by b desc";
+
+            // insert 12 rows into table `test_history_matching_threshold`
+            executeAndTrackHistory("insert into " + tableName + " values(1, '1001'), (2, '1002'), (2, '1003'), (2, '1002'), (2, '1003'), (2, '1002'), (2, '1003'), (2, '1002'), (2, '1003'), (2, '1002'), (2, '1003'), (3, '1004')", defaultSession());
+
+            // cost based statistics
+            assertPlan(query, anyTree(node(TableScanNode.class).withOutputRowCount(false, "CBO")));
+            assertPlan(query, node(OutputNode.class, anyTree(any())).withOutputRowCount(false, "CBO"));
+
+            executeAndTrackHistory(query, defaultSession());
```

Comment:
```suggestion
            executeAndTrackHistory(query, createSessionWithHistoryThresholdAndSkipInputCheck(0.1));
```

Do this change so we can cover the scenario in which input table statistics need to be fetch from metastore when tracking historical statistics after the query's execution.